src/Components/HeroSection.js

```diff
@@ -0,0 +1,61 @@
+import React, {useEffect} from 'react'
+import {Switch, Route} from "react-router-dom"
+import CartList from '../Pages/CartList'
+import ProductsList from '../Pages/ProductsList'
+import ProductDetail from '../Pages/ProductDetail'
+import WishList from '../WishList/WishList'
+import Home from '../Pages/Home'
+import axios from "axios"
+import { useStore } from '../Store/context'
+
+
+const HeroSection = () => {
+    const {dispatch} = useStore()
+    useEffect(() => {
+        async function fetchData() {
+            dispatch({type: "IS_LOADING", payload: "loading"})
+            try {
+                const response = await axios.get("/api/products")
+                const data = await response.data.products
```

Comment:
I am not sure why two await are needed here. 
U can directly pass response.data.products in payload.
Plz correct me if m mistaken !